### PR TITLE
Added http code 424

### PR DESCRIPTION
**Pull request by Arlington1985**

### CODE REVIEW

File: werkzeug/exceptions.py [hunk 1]
```diff
@@ -520,6 +520,20 @@ class Locked(HTTPException):
     )
 
 
+class Depended(HTTPException):
```
Comment (davidism) [inline werkzeug/exceptions.py]: This should match the error: `FailedDependency`

Comment (Arlington1985) [reply werkzeug/exceptions.py]: I can't see such line in the code

Comment (davidism) [reply werkzeug/exceptions.py]: You wrote `Depended`, which doesn't match the exception title, which is "Dependency Failed".

Comment (Arlington1985) [reply werkzeug/exceptions.py]: Ok, got it. Will fix that. Thanks

**Comment by Arlington1985** [comment]
Agreed

File: werkzeug/exceptions.py [hunk 2]
```diff
@@ -520,6 +520,20 @@ class Locked(HTTPException):
     )
 
 
+class FailedDependency(HTTPException):
+
+    """*424* `Failed Dependency`
+
+    Used if the method could not be performed on the resource
```
Comment (lepture) [inline werkzeug/exceptions.py]: There is a trailing whitespace after `resource`, which causes the travis-ci failure.

Comment (Arlington1985) [reply werkzeug/exceptions.py]: Fixed, but still failing

**Pull request merged**